kubernetes/kfserving/examples/v2/bert_handler.py

```diff
@@ -0,0 +1,226 @@
+# !/usr/bin/env/python3
+# Copyright (c) Facebook, Inc. and its affiliates.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# pylint: disable=no-self-use,too-many-arguments,unused-argument,not-callable
+"""Bert Custom Handler."""
```

Comment:
Is there any change between the TorchServe  handler for V1 and V2?

Reply:
Added sample handlers